schedulingApp/unit-tests/test_courses.py

```diff
@@ -1,38 +1,126 @@
 from django.test import TestCase
-
+from models import Course
 
 # Fields: Title
 class CreateCourse(TestCase):
-    def test_emptyTitle(self):
-        pass
+    def test_noArgs(self):
+        a = Course()
+        self.assertRaises(TypeError, msg="Should raise TypeError for no arguments, requires a title and semester")
```

Comment:
The assertRaises doesn't work after the error is raised, proper implementation would look as follows:

```suggestion
        with self.assertRaises(TypeError, msg="Should raise TypeError for no arguments, requires a title and semester"):
                a = Course()
```